Samples/iOS-Swift/iOS-Swift-UITests/UserFeedbackUITests.swift

```diff
@@ -8,14 +6,93 @@ class UserFeedbackUITests: BaseUITest {
     override func setUp() {
         super.setUp()
         app.launchArguments.append(contentsOf: [
-            "--io.sentry.iOS-Swift.auto-inject-user-feedback-widget",
-            "--io.sentry.iOS-Swift.user-feedback.all-defaults",
+            "--disable-spotlight",
+            "--disable-automatic-session-tracking",
+            "--disable-metrickit-integration",
+            "--disable-session-replay",
+            "--disable-watchdog-tracking",
+            "--disable-tracing",
+            "--disable-swizzling",
+            "--disable-network-breadcrumbs",
+            "--disable-core-data-tracing",
+            "--disable-network-tracking",
+            "--disable-uiviewcontroller-tracing",
+            "--disable-automatic-breadcrumbs",
+            "--disable-anr-tracking",
+            "--disable-auto-performance-tracing",
+            "--disable-ui-tracing",
+            "--io.sentry.feedback.auto-inject-widget",
```

Comment:
`m`: Please add the reason why you disable almost everything as a comment. My guess is to reduce flakiness, but I'm not 100%.

Reply:
Yep, that's the reason. I see intermittent crashes from session replay